src/style.css

```diff
@@ -1,72 +1,248 @@
+/* All */
 * {
   box-sizing: border-box;
-  font-family: 'Merriweather', serif;
+  font-family: 'Sono', sans-serif;
 }
 
+/* Body */
 body {
   margin: 0;
   padding: 0;
+  background:
+    linear-gradient(
+      to right,
+      #1e0253 0%,
+      #631447 50%,
+      #fff 50%,
+      #fff 100%
+    );
 }
 
 h1 {
-  font-size: 65px;
-  margin-left: 50px;
+  width: 90%;
+  font-size: 50px;
+  text-align: center;
+  position: relative;
+  left: 75px;
 }
 
 h2 {
-  font-size: 30px;
+  font-size: 25px;
 }
 
+.white {
+  color: #fff;
+}
+
+.black {
+  color: #fff;
+}
+
+/* Main grid */
 .score-cont {
   display: grid;
   grid-template-columns: 1fr 1fr;
-  padding: 0 115px;
+  padding: 0 80px;
   gap: 100px;
 }
 
+/* Leaderboard Section */
 .score-header {
   display: flex;
   justify-content: space-between;
   align-items: center;
 }
 
-.score-header button {
+#refresh {
+  position: relative;
+  top: 5px;
   padding: 5px 15px;
-  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
   background: none;
+  border: none;
   cursor: pointer;
-  border-radius: 5px;
+  -moz-transition: all 0.8s linear;
+  -webkit-transition: all 0.8s linear;
+  transition: all 0.8s linear;
 }
 
-#scores {
-  border: 3.5px solid black;
+#refresh.down {
+  -ms-transform: rotate(180deg);
+  -moz-transform: rotate(180deg);
+  -webkit-transform: rotate(180deg);
+  transform: rotate(180deg);
 }
 
-#scores :nth-child(even) {
-  background-color: #dfdfdf;
+.empty {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  height: 320px;
+  text-align: center;
 }
 
-#scores p {
-  margin: 0;
-  padding: 5px;
+.empty img {
+  height: 100px;
+  width: 100px;
+  margin-bottom: 10px;
+}
+
+.leaderboard {
+  box-shadow: 0 4px 8px 0 rgba(255, 255, 255, 0.2);
+  border-radius: 35px;
+  padding: 20px;
+  height: 370px;
+  background-color: #fff;
+  overflow-y: auto;
+}
```

Comment:
- [ ] The width of the leaderboard is too big, please 🙏🏽 fix it for a better user interface the layout should look like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png).

![image](https://user-images.githubusercontent.com/58642949/202624594-36459045-3d1e-45fb-879f-ba13377ab11d.png)